### PR TITLE
Added Audio for Passing Parameters to a function

### LINKED ISSUE #7221: Audio for passing parameters to the function
### Description

I will do audio contribution on the topic "ways of passing parameters to the function".

### Domain

C/CPP

### Type of Contribution

Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Harsh652-cpu**
<hr>

## Description 📜

In a function we have to pass arguments from calling function to called function.But to do that we have two methods.One is call by value and another one is call by reference.Both are having its own advantages and disadvantages at specific conditions.In this audio file I have explained all those points which are important to understand this topic with proper examples.I hope it will add some values in your coding journey!
Fixes #7221 

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Harsh652-cpu** [comment]
@Saviour1001  @Suvraneel @apeksha235 please check it.

File: C_CPP/Fundamentals/Functions & Recursion and its types/Audio_passing_parameters_to_a_function.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+*In this audio file,I have covered following topics:-*
+1. What is calling function and called function?
+2. What is the difference between a calling function and a called fucntion?
+3. How we can pass arguments from calling function to called function?In how many ways?
+4. What is call by value?How it is used while passing paramters to a function?What are its advantages and disadvantages?
+5. What is call by reference?Why and in which condition we use call by reference instead of using call by value method?
+6. Some examples based on both call by value and call by reference.
+
+## Audio file
+[Audio for passing parameters to a function](https://drive.google.com/file/d/1NAatKPELWlApVAOIkjzf-FwkwBbdTKZn/view?usp=sharing)
```
Comment (apeksha235) [inline C_CPP/Fundamentals/Functions & Recursion and its types/Audio_passing_parameters_to_a_function.md:12]: @Harsh652-cpu dictating the code in your audio is not a good practice, please mention only the major syntax necessary and explain the rest in words

**Review by apeksha235 — CHANGES_REQUESTED** [review]

File: C_CPP/Fundamentals/Functions & Recursion and its types/Audio_passing_parameters_to_a_function.md (hunk 1 above)
Comment (Harsh652-cpu) [reply C_CPP/Fundamentals/Functions & Recursion and its types/Audio_passing_parameters_to_a_function.md]: Done @apeksha235

**Comment by Harsh652-cpu** [comment]
I have created a new audio and added it in this pr.Please review it now.

**Review by vasu-1 — APPROVED** [review]
LGMT @Harsh652-cpu 

@Suvraneel

**Pull request merged**